app/routes/users.js

```diff
@@ -4,22 +4,37 @@ const userService = require('../services').userService;
 
 /* GET users listing. */
 router.get('/', function(req, res, next) {
-  res.send('respond with a resource');
+  return req.sessionID ? res.redirect('/#!/signin') : res.redirect('/#!/signup');
```

Comment:
Created middleware inside directory blogger/app/lib. Naming may need to be changed but introduced for GET requests to /users route.